adopters/organizations/halodoc.md

```diff
@@ -0,0 +1,32 @@
+## **Halodoc**
+
+[Halodoc](https://www.halodoc.com/) is a secure health-tech platform with a mission to simplifying access to healthcare by connecting millions of patients with licensed doctors, insurance, labs, and pharmacies in one simple mobile application. Halodoc’s innovative technology, nimble services, and patient focus enable a host of solutions including 24/7 doctor tele consultation via chat, voice or video; medicine purchase & delivery; lab services at home; and strong customer support.
+
+Halodoc is the 2018 Forbes Indonesia Choice Award winner and Galen Growth’s 2018 Most Innovative HealthTech Startup in Asia, a testimony to a team of compassionate, innovative, trustworthy and agile people who take ownership of their work in building the most trusted digital healthcare company. 
+
+### **Why we explored litmus**
+
+We wanted a tool that we could leverage to test the resiliciency of multi k8s cluster workloads in private cloud, 
```

Comment:
```suggestion
We wanted a tool that we could leverage to test the resiliency of multi k8s cluster workloads in a private cloud, 
```

Reply:
updated spell resiliency and "in a private"